polkadot-protocol/smart-contract-basics/networks.md

```diff
@@ -0,0 +1,67 @@
+
+2. **Westend Asset Hub** - after testing locally, contracts are deployed to Westend Asset Hub, the primary TestNet for smart contract development. It simulates real-world conditions without using real tokens, making it ideal for rigorous contract testing
+
+3. **Kusama Asset Hub** - once contracts have passed extensive testing on Westend, they can be deployed to Kusama Asset Hub, the "canary" network version. This provides a high-fidelity testing ground with actual economic incentives
+
+4. **Polkadot Asset Hub** - after passing tests on Westend and Kusama Asset Hub, smart contracts are considered ready for deployment to Polkadot Asset Hub, the live production network
+
+In addition, developers can leverage the upcoming Paseo Asset Hub for additional testing capabilities:
+
+- **Paseo Asset Hub** - Serves as a community-run TestNet that mirrors Asset Hub's runtime. Like Westend Asset Hub for protocol development, Paseo provides a testing ground for smart contract development without affecting live networks
+
+## Local Development
+
+The local development environment is crucial for smart contract development on Asset Hub. It provides developers a controlled space for rapid testing and iteration before moving to public networks. The local setup consists of several key components:
+
+- [**Kitchensink node**](https://paritytech.github.io/polkadot-sdk/master/pallet_revive_eth_rpc/subxt_client/src_chain/runtime_types/kitchensink_runtime/index.html){taget=\_blank} - a local node that can be run for development and testing. It includes logging capabilities for debugging contract execution and provides a pre-configured development environment with pre-funded accounts for testing purposes
+- [**Ethereum RPC proxy**](https://paritytech.github.io/polkadot-sdk/master/pallet_revive_eth_rpc/index.html){target=\_blank} - bridges Ethereum-compatible tools with the Polkadot SDK-based network. It enables seamless integration with popular development tools like MetaMask and Remix IDE. The purpose of this component is to translate Ethereum RPC calls into Substrate format
+
+## Test Networks
+
+The following test networks provide controlled environments for testing smart contracts. TestNet tokens are available from the [Polkadot faucet](https://faucet.polkadot.io/){target=\_blank}.
+
+### Westend Asset Hub
+
+Westend Asset Hub is the primary permanent TestNet for smart contract development. It provides a stable environment for testing contracts without using real tokens. The network maintains the same features and capabilities as the production Asset Hub, making it an ideal environment for thorough testing before moving to production networks.
+
+### Paseo Asset Hub
+
+The upcoming Paseo Asset Hub will serve as a community-managed TestNet specifically designed for smart contract development. It will mirror Asset Hub's runtime and provide developers with an additional environment for testing their contracts before deployment to production networks.
```

Comment:
```suggestion
The upcoming Paseo Asset Hub will be a community-managed TestNet designed specifically for smart contract development. It will mirror Asset Hub's runtime and provide developers with an additional environment for testing their contracts before deployment to production networks.
```